### PR TITLE
Assigment 2

### PR DESCRIPTION
**Pull request by JoseDelPino**
Here are my three files for the second assignment

### CODE REVIEW

File: Assignment2/JoseDelPino-JoseDelPino.ttl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Class01 :includes :Computer101,
+        :Sensor029 .
+
+:Computer101 :hasOwner :User10A .
+
+:Measurement801 :atTime "2010-06-12T12:00:12" ;
```
Comment (ocorcho) [inline Assignment2/JoseDelPino-JoseDelPino.ttl]: Missing datatypes here and in the following line

File: Assignment2/JoseDelPino-JoseDelPino.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <includes rdf:resource="#Sensor029" />
+    <includes rdf:resource="#Computer101" />
+</rdf:Description>
+
+<rdf:Description rdf:about="#Sensor029">
+    <hasMeasurement rdf:resource="#Measurement801" />
+</rdf:Description>
+
+<rdf:Description rdf:about="#Measurement801">
+    <hasTemperature>29</hasTemperature>
```
Comment (ocorcho) [inline Assignment2/JoseDelPino-JoseDelPino.xml]: Same here with datatypes

**Comment by ocorcho** [comment]
check my comments

**Comment by JoseDelPino** [comment]
> check my comments

Sorry, I thought they were just indications. Should I change them to the correct version?

**Pull request closed without merging**